### DISCUSSION TITLE
how can response image as API output?

### DISCUSSION THREAD

**Discussion by aliebrahiiimi** [bentoml/BentoML#2345] [Q&A]
Hi, I produce an image.png in my code and want to respond it as my API output, but I can't find a good example for that, could you help me, please?
I use bentoml==0.13

**Comment by timliubentoml** [comment]
Hello! Apologies that it took some time to get back. I think we missed this. If you look here at the output adapters: https://docs.bentoml.org/en/0.13-lts/api/adapters.html?highlight=adapter#jsonoutput

You can see the output types. Unfortunately the best way that I can think of is to use a JsonOutput and return the image in base64
